deploy/docker-compose-build.yml

```diff
@@ -40,31 +43,32 @@ services:
     environment:
       BCRYPT_SALT: ${BCRYPT_SALT}
       SESSION_EXPIRATION_IN_MINUTES: ${SESSION_EXPIRATION_IN_MINUTES}
-      DB_URL: postgres://${DB_USER}:${DB_PASSWORD}@ballerine-postgres:${DB_PORT}
+      DB_URL: postgres://${DB_USER}:${DB_PASSWORD}@postgres:${DB_PORT}
       API_KEY: ${API_KEY}
       NODE_ENV: ${NODE_ENV}
       COMPOSE_PROJECT_NAME: ${COMPOSE_PROJECT_NAME}
       DB_PORT: ${DB_PORT}
       DB_USER: ${DB_USER}
       DB_PASSWORD: ${DB_PASSWORD}
       SESSION_SECRET: ${SESSION_SECRET}
-      BACKOFFICE_CORS_ORIGIN: http://localhost:${BACKOFFICE_PORT}
-      WORKFLOW_DASHBOARD_CORS_ORIGIN: http://localhost:${WORKFLOW_DASHBOARD_PORT}
+      BACKOFFICE_CORS_ORIGIN: http://${DOMAIN_NAME}:${BACKOFFICE_PORT}
```

Comment:
In this case if the domain name id undefined, it should probably fallback to localhost